delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -463,32 +498,34 @@ def checkServiceExists(service_name, domain, region) {
 		} else if(resultJson.data.available == true) {
 			error "No service exists with name "+service_name+" under domain "+domain
 		}
-		//events.sendCompletedEvent('VALIDATE_PRE_BUILD_CONF', "Service exists for deletion")
+		events.sendCompletedEvent('VALIDATE_PRE_BUILD_CONF', "Service exists for deletion")
 	}catch (ex) {
 		if(!((ex.getMessage()).indexOf("groovy.json.internal.LazyMap") > -1)) {
 			updateServiceInDB(var_db_service_id, var_api_id_dev, region, "deletion_failed");
 			send_status_email ("FAILED")
+            events.sendFailureEvent('VALIDATE_PRE_BUILD_CONF', "service deletion failed")
 			error "checkServiceExists Failed. "+ex.getMessage()
 		} else {
-			//events.sendCompletedEvent('VALIDATE_PRE_BUILD_CONF', "Service exists for deletion")
+			events.sendCompletedEvent('VALIDATE_PRE_BUILD_CONF', "Service exists for deletion")
 		}
 	}
 }
 
 
 def deleteserviceindb(region, db_id) {
-	//events.sendStartedEvent('VALIDATE_PRE_BUILD_CONF', "Check if service exists")
+	events.sendStartedEvent('VALIDATE_PRE_BUILD_CONF', "Check if service exists")
 	try {
 		sh "curl -X DELETE -k -v -H \"Authorization: ${params.auth_token}\" \
 				'https://${var_api_id_dev}.execute-api.${region}.amazonaws.com/dev/platform/services/${db_id}'"
-		//events.sendCompletedEvent('VALIDATE_PRE_BUILD_CONF', "Service exists for deletion")
+		events.sendCompletedEvent('VALIDATE_PRE_BUILD_CONF', "Service exists for deletion")
 	}catch (ex) {
 		if(!((ex.getMessage()).indexOf("groovy.json.internal.LazyMap") > -1)) {
 			updateServiceInDB(var_db_service_id, var_api_id_dev, region, "deletion_failed");
 			send_status_email ("FAILED")
+            events.sendFailureEvent('VALIDATE_PRE_BUILD_CONF', "service deletion failed")
 			error "deleteService Failed. "+ex.getMessage()
 		} else {
-			//events.sendCompletedEvent('VALIDATE_PRE_BUILD_CONF', "Service exists for deletion")
+			events.sendCompletedEvent('VALIDATE_PRE_BUILD_CONF', "Service exists for deletion")
```

Comment:
You are still in catch, how can you send a completed event just because you dont have an exception message!

Reply:
Thats true. Changed this to send a send failure event

Reply:
Rethinking on this, was there a scenario where exception was thrown but still we needed to go on with the flow and thats why the send completed event was added but not the 'error' was not added after complete event.